### DISCUSSION TITLE
How to install anything to Podman OS X virtual machine?

### DISCUSSION THREAD

**Discussion by Sitin** [containers/podman#16080] [General]
The machine used in Podman for OS X is missing package managers. How can I install any required software?

```shell
[core@localhost ~]$ which yum
/usr/bin/which: no yum in (/var/home/core/.local/bin:/var/home/core/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin)
```

```shell
[core@localhost ~]$ which dnf
/usr/bin/which: no dnf in (/var/home/core/.local/bin:/var/home/core/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin)
```

Looks like only RPM is available:

```shell
[core@localhost ~]$ which rpm
/usr/bin/rpm
```

**Comment by rhatdan** [comment]
The system is using rpm-ostree?  What software do you want to install?  We usually just launch containers within the VM and install software via the containers.

**Reply by dhduvall** [reply]
I'm running a shell inside the VM because I want to be able to do `podman build -v ...`, which isn't supported when running `podman` outside the VM. Since I'm actually spending some quality time there, I'd love to be able to install some creature comforts like `zsh`, `less`, etc.

**Reply by baude** [reply]
rpm-ostree is perfectly capable of being able to add software ... i dont recall the exact commands offhand but if you cannot figure it out, hit us up here again.

**Comment by afbjorklund** [comment]
I think it was _supposed_ to be immutable, i.e. it's a OS feature

https://fedoramagazine.org/introducing-fedora-coreos/
